products/installation.yaml

```diff
@@ -105,7 +105,7 @@ products:
     channel: alpha
     package: observability-operator
     installFrom: "index"
-    index: "quay.io/rhoas/observability-operator-index:v3.0.8"
+    index: "quay.io/rhoas/observability-operator-index:v3.0.9"
```

Comment:
Think you need to update the cpaas file as well https://github.com/integr8ly/integreatly-operator/blob/master/products/installation-cpaas.yaml

Reply:
updated. 👍